components/sysutils/mc/patches/04-clarify_sighandler_t_usage_in_non-GNU_OSes.patch

```diff
@@ -0,0 +1,46 @@
+https://github.com/MidnightCommander/mc/commit/4f06ba93460751bd979ca5cc87a49219b1f78a9f
+
+From 4f06ba93460751bd979ca5cc87a49219b1f78a9f Mon Sep 17 00:00:00 2001
+From: Andrew Borodin <aborodin@vmail.ru>
+Date: Fri, 22 Feb 2019 11:32:43 +0300
+Subject: [PATCH] Ticket #3971: clarify sighandler_t usage in non-GNU OSes.
+
+Signed-off-by: Andrew Borodin <aborodin@vmail.ru>
+---
+ m4.include/mc-tests.m4                 | 6 ++++++
+ tests/lib/utilunix__my_system-common.c | 5 +++++
+ 2 files changed, 11 insertions(+)
+
+diff --git a/m4.include/mc-tests.m4 b/m4.include/mc-tests.m4
+index 95e948bd7f..257218ecbe 100644
+--- a/m4.include/mc-tests.m4
```

Comment:
Do I miss something? It seems you patch m4 file, but do not regenerate configure.

Reply:
@Mno-hime

Reply:
Right... this and that the `test` target is actually starting any tests even without the patch. Ah. Lets do it with the next `mc` release.